cli/tests/unit_node/crypto/crypto_key_test.ts

```diff
@@ -202,3 +203,28 @@ for (const primeLength of [1024, 2048, 4096]) {
     },
   });
 }
+
+const rsaPrivateKey = Deno.readTextFileSync(
+  new URL("../testdata/rsa_private.pem", import.meta.url),
```

Comment:
This could be `import.meta.resolve("../testdata/rsa_private.pem");`

Reply:
`Deno.readTextFileSync` does not support the `file://` scheme